source/plugins/projects/projects/index.js

```diff
@@ -973,7 +973,7 @@ let projects = {
 	 * 
 	 * @example
 	 * 
-	 * deleteFile('MyNewProject', 'C:\Users\User\Desktop\file');
+	 * deleteFile(project, 'C:\Users\User\Desktop\file');
```

Comment:
The second parameter is a folder within the project, it cannot be with a full path, it has to start with /...

Reply:
The path still starts with `C:\...`